src/Builder/ListBuilder.php

```diff
@@ -57,7 +58,7 @@ public function buildField($type, FieldDescriptionInterface $fieldDescription, A
     {
         if (null == $type) {
             $guessType = $this->guesser->guessType($admin->getClass(), $fieldDescription->getName(), $admin->getModelManager());
-            $fieldDescription->setType($guessType->getType());
+            $fieldDescription->setType(($guessType instanceof TypeGuess ? $guessType->getType() : null));
```

Comment:
are the additional brackets needed? 🤔 

Or does this work, too?:
```suggestion
            $fieldDescription->setType($guessType instanceof TypeGuess ? $guessType->getType() : null);
```

Reply:
It definitely should

Reply:
in free living conditions i would add them for better readability. Here it is an edge case i would say, it should work at the end, but i will fix it.
btw: you know a good boolean cast in javascript? `!!(condition)` *g* that is quite awesome writing angular expressions.